package.json

```
@@ -1,35 +1,42 @@
 {
   "name": "level-errors",
-  "description": "Error types for LevelUP",
   "version": "2.0.0",
+  "description": "Error types for LevelUP",
+  "license": "MIT",
+  "main": "errors.js",
+  "scripts": {
+    "test": "standard && hallmark && nyc node test.js",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "hallmark": "hallmark --fix"
+  },
+  "dependencies": {
+    "errno": "~0.1.1"
+  },
+  "devDependencies": {
+    "coveralls": "^3.0.2",
+    "hallmark": "^0.1.0",
+    "level-community": "^3.0.0",
+    "nyc": "^12.0.2",
+    "standard": "^12.0.0",
+    "tape": "^4.3.0"
+  },
+  "hallmark": {
+    "community": "level-community"
+  },
+  "typings": "errors.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/level/errors.git"
```

```suggestion
     "url": "https://github.com/Level/errors.git"
```

(Some tools out there are case-sensitive)